containers/proxy-systemd-services/uyuni-proxy-httpd.service

```diff
@@ -15,7 +15,13 @@ EnvironmentFile=/etc/sysconfig/uyuni-proxy-systemd-services
 EnvironmentFile=/etc/sysconfig/proxy
 Restart=on-failure
 ExecStartPre=/bin/rm -f %t/uyuni-proxy-httpd.pid %t/uyuni-proxy-httpd.ctr-id
-ExecStart=/usr/bin/podman run --conmon-pidfile %t/uyuni-proxy-httpd.pid --cidfile %t/uyuni-proxy-httpd.ctr-id --cgroups=no-conmon --pod-id-file %t/uyuni-proxy-pod.pod-id -d --replace -dt -v /etc/uyuni/proxy:/etc/uyuni:ro -v uyuni-proxy-rhn-cache:/var/cache/rhn -v uyuni-proxy-tftpboot:/srv/tftpboot --name uyuni-proxy-httpd ${NAMESPACE}/proxy-httpd:${TAG}
+ExecStart=/usr/bin/podman run \
+    --conmon-pidfile %t/uyuni-proxy-httpd.pid \
+    --cidfile %t/uyuni-proxy-httpd.ctr-id \
+    --cgroups=no-conmon \
+    --pod-id-file %t/uyuni-proxy-pod.pod-id -d \
+    --replace -dt -v /etc/uyuni/proxy:/etc/uyuni:ro -v uyuni-proxy-rhn-cache:/var/cache/rhn -v uyuni-proxy-tftpboot:/srv/tftpboot \
+    --name uyuni-proxy-httpd ${HTTPD_NAMESPACE:=NAMESPACE}/proxy-httpd:${HTTPD_TAG:=TAG}
```

Comment:
Did you try if this value assignment works as a fallback from HTTPD_TAG to TAG (for example) as expected?
The syntax looks strange and I made a quick test locally and it didn't look it fall back.

Reply:
Yes, it's part of the [parameter expansion](https://www.gnu.org/software/bash/manual/html_node/Shell-Parameter-Expansion.html) (if I read the docs correctly), and it worked at first glance. Let me test it again to see if I missed something.

Reply:
it may very well work different in Zsh though

Reply:
a systemd unit should not depend on particular shell features...